blog/index.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Refuge Restrooms
+---
+<div>
+	<h1>Welcome to Our Blog</h1>
+	<p>REFUGE restrooms indexes and maps safe restroom locations for trans, intersex, and gender nonconforming individuals.</p>
+	<ul class="posts">
+	  {% for post in site.posts %}
+	    <li><span>{{ post.date | date_to_string }}</span> » <a href="/refugerestrooms{{ post.url }}" title="{ post.title }}">{{ post.title }}</a></li>
```

Comment:
There is apparently a missing left bracket here:

`title="{ post.title }}"` --> `title="{{ post.title }}"`

Reply:
I'll fix these as soon as possible.